terraform/projects/app-ecs-instances/main.tf

```diff
@@ -47,6 +41,10 @@ variable "ecs_instance_ssh_keyname" {
   default     = "ecs-monitoring-ssh-test"
 }
 
+variable "ecs_optimised_ami_version" {
```

Comment:
This move to a version number rather than an AMI version, does it mean that the README instructions on finding the latest AMI ID is no longer so useful? (I don't know the answer)

Reply:
README should probably be updated? Will check with Ed